pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "ansys-templates"
-version = "0.4.dev0"
+version = "0.4.1.dev0"
```

Comment:
No need to update the version. As soon as this PR gets merged, a new patch release will be made to make these changes available to public.